webpack/clientNoReactNoMinim.js

```diff
@@ -0,0 +1,50 @@
+import merge from 'lodash/merge';
+
+import clientWebpackConfig from './client';
+
+export default merge({}, clientWebpackConfig, {
+  output: {
+    filename: 'attributes-kit-no-react-no-minim.js',
+  },
+
+  externals: {
+    'react': {
+      root: 'React',
+      commonjs2: 'react',
+      commonjs: 'react',
+      amd: 'react'
+    },
+    'react-dom': {
+      root: 'react-dom',
+      commonjs2: 'react-dom',
+      commonjs: 'react-dom',
+      amd: 'react-dom'
+    },
+    'prop-types': {
+      root: 'prop-types',
+      commonjs2: 'prop-types',
+      commonjs: 'prop-types',
+      amd: 'prop-types'
+    },
+    'minim': {
+      root: 'minim',
+      commonjs2: 'minim',
+      commonjs: 'minim',
+      amd: 'minim'
+    },
+    'minim/lib/serialisers/json-0.6': {
+      root: 'minim/lib/serialisers/json-0.6',
+      commonjs2: 'minim/lib/serialisers/json-0.6',
+      commonjs: 'minim/lib/serialisers/json-0.6',
+      amd: 'minim/lib/serialisers/json-0.6'
+    },
+    'minim-parse-result': {
```

Comment:
I don't fully understand whats going on with web pack, but I don't think this is needed as `minim-parse-result` namespace is not needed providing you use the fury minim provided namespace (in comment https://github.com/apiaryio/attributes-kit/pull/462/files#r192442257)

Reply:
Thanks, dropped, using `fury.minim`